liver_imaging_analysis/engine/engine.py

```diff
@@ -535,6 +587,64 @@ def predict(self, data_dir):
         return prediction_list
 
 
+    def automate(self,optimization_direction,epochs,evaluate_epochs,batch_callback_epochs,save_weight,save_path,trial_numbers):
```

Comment:
I think tune_parameters() would be a more descriptive name for this method than automate.

Reply:
okay